VideoWeb/VideoWeb/ClientApp/src/app/testing/mocks/data/conference-test-data.ts

```diff
@@ -403,31 +403,31 @@ export class ConferenceTestData {
             id: Guid.create().toString(),
             from: 'vho.user@hearings.net',
             message: 'test message from vho',
-            timestamp: moment(now).subtract(3, 'minutes').toDate()
+            timestamp: moment(now).subtract(3, 'minutes').toDate(),
```

Comment:
There are no changes here. Could you revert this file please.

Reply:
done